### DISCUSSION TITLE
Password construction, boundary conditions of the password for logging into a GitHub account?

### DISCUSSION THREAD

**Discussion by Pytek2023** [community/community#65826] [Other Feature Feedback, Questions, & Ideas]
Write Bing Search Assistant what are the conditions for a password on GitHub with a lowercase letter, an uppercase letter, a number, a special character in addition to this condition that it has to be at least 15 characters long. Which characters from a QWERTY keyboard cannot be used in a password for logging into a GitHub account? 
Bing Search Assistant answers: "Yes, I am sure that all special characters available on the QWERTY keyboard can be used to build a password for logging into a GitHub account. I have not found any information on prohibited characters in the password for logging into a GitHub account. It seems that there are no restrictions on using special characters from the QWERTY keyboard in your GitHub account login password. It is important that your password is unique and difficult to guess." 
True or false?

**Comment by jesse-sealand** [comment]
True: According to GitHub documentation.

"You must choose or generate a password for your account on GitHub.com that is at least:

Eight characters long, if it includes a number and a lowercase letter, or
15 characters long with any combination of characters
""
[Source Page](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-strong-password)
